components/Chat.js

```diff
@@ -109,6 +130,10 @@ export default memo(function Chat({
       ref={scrollRef}
       getItemLayout={getItemLayout}
       scrollEventThrottle={1000}
+      // This is deprecated so it'll have to be fixed
+      // (the issue of not all elements being visible when keybaord is up and message view is more than 3 lines)
+      // https://github.com/atharesinc/athares/pull/13
+      disableVirtualization={isKeyboardVisible}
```

Comment:
This will prevent the weird behavior due to virtualization when the user it typing. In theory, it prevents unmounting of unseen items and mounting of items yet to be seen, so the user only has which items have been loaded. So it's less performant, but only when the user isn't likely to be scrolling through stuff.  Some alternative solution should be resolved before it just breaks.